CHANGELOG.md

```diff
@@ -1,8 +1,78 @@
 # `chainweb-node` Changelog
 
+## 2.23 (2023-02-XX)
+This version replaces all previous versions. Any prior version will stop working
+on **2024-03-06T00:00:00Z**. Node administrators must upgrade to this version before
+that date.
+
+This version will expire on TODO FIXME.
+
+To upgrade, pull the latest docker image or download the binary and restart the node.
+
+Changes:
+- Updated to Pact 4.11 (TODO FIXME LINK)
+- The `coin` contract was updated to version 6, implementing KIP-0022 (#1807)
+- Pact "verifier plugins" are now available, implementing KIP-0028 (#1777)
+- Fix a bug where nodes could take too long to start up if they took too long to
+  get the latest cut. (#1791)
+- Rename two user-visible network IDs (#1810)
+  - `development` was renamed to `recap-development`
+  - `fast-development` was renamed to `development`
+- Running a node and making queries to `/local` should be much faster when the
+  `?rewindDepth` query parameter is provided.
+- New REST API endpoint features (#1800):
+  - The `/payload` GET endpoint now:
+    - Takes a `?height` query parameter, allowing you to specify the block
+      height of the payload you are querying.
+    - You can now do a "batch query" by submitting multiple hashes and heights at
+      the same time. Instead of submitting a list of hashes, submit a JSON object
+      such as:
+
+      ```
+      { "hashes": ["hash1", "hash2", "hash3"], "heights": [1, 2, 3] }
+      ```
+
+      See the Chainweb OpenAPI specification for more information on how to use
+      this feature.
```

Comment:
Actually we could already do this, the new thing is just the height specification (which also will be mandatory in future)